src/core/api.rs

```diff
@@ -279,22 +289,30 @@ impl EncryptedHeader {
         cover_crypt: &Covercrypt,
         policy: &Policy,
         public_key: &MasterPublicKey,
-        encryption_policy: &AccessPolicy,
+        encryption_policy: &str,
```

Comment:
I thought you were in favor of reverting the move from `AccessPolicy` to `String`?